src/main/scala/me/shadaj/scalapy/tensorflow/keras/layers/Layers.scala

```diff
@@ -0,0 +1,49 @@
+package me.shadaj.scalapy.tensorflow.keras.layers
+
+import me.shadaj.scalapy.py
+import me.shadaj.scalapy.tensorflow.Tensor
+
+@py.native trait Layers extends py.Object {
+  def Conv2D(filters: Int,
+             kernel_size: py.|[Int, (Int, Int)],
+             strides: py.|[Int, (Int, Int)] = (1, 1),
+             padding: String = "valid",
+             data_format: py.NoneOr[String] = py.None,
+             dilation_rate: py.|[Int, (Int, Int)] = (1, 1),
+             activation: py.NoneOr[String] = py.None,
+             use_bias: Boolean = true,
+             kernel_initializer: String = "glorot_uniform",
+             bias_initializer: String = "zeros",
+             kernel_regularizer: py.NoneOr[String] = py.None,
+             bias_regularizer: py.NoneOr[String] = py.None,
+             activity_regularizer: py.NoneOr[String] = py.None,
+             kernel_constraint: py.NoneOr[String] = py.None,
+             bias_constraint: py.NoneOr[String] = py.None,
+             input_shape: py.NoneOr[(Int, Int, Int)] = py.None
```

Comment:
`input_shape` should be provided via `**kwargs` (https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D) but I didn't know how to do it with `scalapy`

Reply:
Hmm, this is something I need to work on supporting, probably with an annotation. In the meantime, you should be able to handle this by calling `this.as[py.Dynamic].Conv2D(..., input_shape=input_shape, ...)` since the dynamic API does support kwargs.

Reply:
I tried the following in dynamic API:
```scala
val models2 = py.module("tensorflow").keras.models
val layers2 = py.module("tensorflow").keras.layers

val model = models2.Sequential() // kerasA.models.as[py.Dynamic].Sequential() // kerasA.models.Sequential()
model.add(layers2.Conv2D(filters = 32, kernel_size = (3, 3), activation = "relu", input_shape = (28, 28, 1)))
```

And it fails with:

```
[error] Exception in thread "main" me.shadaj.scalapy.py.PythonException: <class 'TypeError'> Dimension value must be integer or None or have an __index__ method, got 32.0
[error] 	at me.shadaj.scalapy.py.CPythonInterpreter$.$anonfun$throwErrorIfOccured$1(CPythonInterpreter.scala:110)
[error] 	at me.shadaj.scalapy.py.Platform$.Zone(Platform.scala:10)
[error] 	at me.shadaj.scalapy.py.CPythonInterpreter$.throwErrorIfOccured(CPythonInterpreter.scala:96)
[error] 	at me.shadaj.scalapy.py.CPythonInterpreter$.$anonfun$load$1(CPythonInterpreter.scala:119)
[error] 	at me.shadaj.scalapy.py.Platform$.Zone(Platform.scala:10)
[error] 	at me.shadaj.scalapy.py.CPythonInterpreter$.load(CPythonInterpreter.scala:116)
[error] 	at me.shadaj.scalapy.py.package$.eval(package.scala:84)
[error] 	at me.shadaj.scalapy.py.AnyDynamics.applyDynamicNamed(Dynamic.scala:17)
[error] 	at me.shadaj.scalapy.py.AnyDynamics.applyDynamicNamed$(Dynamic.scala:14)
[error] 	at me.shadaj.scalapy.py.package$$anon$11$$anon$12.applyDynamicNamed(package.scala:84)
[error] 	at me.shadaj.scalapy.py.AnyDynamics.applyDynamic(Dynamic.scala:11)
[error] 	at me.shadaj.scalapy.py.AnyDynamics.applyDynamic$(Dynamic.scala:8)
[error] 	at me.shadaj.scalapy.py.package$$anon$11$$anon$12.applyDynamic(package.scala:84)
[error] 	at me.shadaj.scalapy.tensorflow.MnistExample$.delayedEndpoint$me$shadaj$scalapy$tensorflow$MnistExample$1(MnistExample.scala:56)
[error] 	at me.shadaj.scalapy.tensorflow.MnistExample$delayedInit$body.apply(MnistExample.scala:10)
[error] 	at scala.Function0.apply$mcV$sp(Function0.scala:39)
[error] 	at scala.Function0.apply$mcV$sp$(Function0.scala:39)
[error] 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
[error] 	at scala.App.$anonfun$main$1$adapted(App.scala:80)
[error] 	at scala.collection.immutable.List.foreach(List.scala:392)
[error] 	at scala.App.main(App.scala:80)
[error] 	at scala.App.main$(App.scala:78)
[error] 	at me.shadaj.scalapy.tensorflow.MnistExample$.main(MnistExample.scala:10)
[error] 	at me.shadaj.scalapy.tensorflow.MnistExample.main(MnistExample.scala)
```

It casts `32` to `Double` somewhere. I didn't have this issue in previous versions (with JEP) so perhaps there is a bug on master?